pandas/_libs/tslibs/src/datetime/np_datetime.c

```diff
@@ -682,19 +682,23 @@ void pandas_timedelta_to_timedeltastruct(npy_timedelta td,
     npy_int64 sfrac;
     npy_int64 ifrac;
     int sign;
-    npy_int64 DAY_NS = 86400000000000LL;
+    npy_int64 PER_DAY;
```

Comment:
Just as a matter of convention should probably be lowercase. Usually you uppercase macros and constants, so on first read looking at these in the body of the functions a reader may mistakenly think that (DAY_NS was incorrect)

Reply:
good to know, will update